doc/source/programs/gdal_raster_scale.rst

```diff
@@ -0,0 +1,103 @@
+.. _gdal_raster_scale_subcommand:
+
+================================================================================
+"gdal raster scale" sub-command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Scale the values of the bands of a raster dataset
+
+.. Index:: gdal raster scale
+
+Synopsis
+--------
+
+.. program-output:: gdal raster scale --help-doc
+
+Description
+-----------
+
+:program:`gdal raster scale` can be used to rescale the input pixels values
+from the range :option:`--srcmin` to :option:`--srcmax` to the range
+:option:`--dstmin` to :option:`--dstmax`.
+It is also often necessary to reset the output datatype with the :option:`--ot` switch.
+If omitted the output range is 0 to 255.
+If omitted the input range is automatically computed from the source dataset.
+This may be a slow operation on a large source dataset, and if using it multiple times
+for several gdal_translate invocation, it might be beneficial to call
+``gdal raster info --stats {source_dataset}`` priorly to precompute statistics, for
+formats that support serializing statistics computations (GeoTIFF, VRT...)
+Note that the values specified for the ranges are only used to compute a scale and
+offset to apply to the input raster values. In particular, ``srcmin`` and
+``srcmax`` are not used to clip input values unless :option:`--exponent`
+is also specified.
+Instead of being clipped, source values that are outside the range of ``srcmin``
+and ``srcmax`` will be scaled to values outside the range of ``dstmin`` and ``dstmax``.
+If clipping without exponential scaling is desired,
+``--exponent 1`` can be used.
+
+By default, the scaling is applied to all bands. It is possible to restrict
+it to a single band with :option:`--band` and let values of other bands unmodified.
+
+This command is the reverse operation of :ref:`gdal_raster_unscale_subcommand`.
+
+Standard options
+++++++++++++++++
+
+.. include:: gdal_options/of_raster_create_copy.rst
+
+.. include:: gdal_options/co.rst
+
+.. include:: gdal_options/overwrite.rst
+
+.. option:: --ot, --datatype, --output-data-type <OUTPUT-DATA-TYPE>
+
+  Output data type among ``Byte``, ``Int8``, ``UInt16``, ``Int16``, ``UInt32``,
+  ``Int32``, ``UInt64``, ``Int64``, ``CInt16``, ``CInt32``, ``Float32``,
+  ``Float64``, ``CFloat32``, ``CFloat64``.
+
+.. option:: -b, --band <BAND>
+
+    Index (starting at 1) of the band to which the scaling must be only applied.
+
+.. option:: --srcmin <SRCMIN>
+
+    Minimum value of the source range. This option must be used together with :option:`--srcmax`.
```

Comment:
```suggestion
    Minimum value of the source range. If not specified, it will be calculated from the input dataset. This option must be used together with :option:`--srcmax`.
```

Reply:
fixed